docker-testing/contents/datalake-datastore/test.py

```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/python
+
+import os
+import subprocess
+import sys
+
+sys.path.append("..")
+from  run_command import run_command
+
+def main():
+
+    db_args = f'--db_args="host={os.environ["NETMELD_DB_HOST"]} user={os.environ["NETMELD_DB_USER"]} password={os.environ["NETMELD_DB_PASS"]}"'
+    db_name = f'--db_name="{os.environ["NETMELD_DB"]}"'
```

Comment:
I didn't see anything in particular which you are doing that requires this to be defined.  The tools provide their own default.  For this Issue/PR, I don't think we need to support supplying custom parameters.
NOTE: This will require removal of these arguments from all following commands.
```suggestion
```